openvm/tests/apc_builder_single_instructions.rs

```diff
@@ -10,12 +10,12 @@ fn assert_machine_output(program: Vec<Instruction<BabyBear>>, test_name: &str) {
 
 // ALU Chip instructions
 #[test]
-fn single_add_0() {
```

Comment:
This test was kind of redundant with what I renamed to to `load_immediate` above. Adding 1, however, tests a common pattern and is a good example of #3366.